.gitattributes

```diff
@@ -0,0 +1,37 @@
+# Enable automatic line ending conversion for text files on checkout and commit.
+# For all extensions that are not being explicitly defined below git will try to guess
+# whether file contains text and if so, will convert the line endings.
+* text=auto
+
+# Define all the file extensions for which we should always apply line ending covnersion
+*.cs      text diff=csharp
```

Comment:
`text diff=csharp` looks pretty cool 👍 What about the other extensions? `.fs`, `.xml`, etc? 😃

Reply:
`csharp` is the only supported language we need. See the full list [here](https://git-scm.com/docs/gitattributes#_defining_a_custom_hunk_header).

Reply:
I'm not surprised that I don't see `fsharp` on that list. FWIW, AutoFixture.AutoFoq and Idioms.FsCheck are written [*in*<sup>*</sup>](https://codeblog.jonskeet.uk/2008/04/23/programming-quot-in-quot-a-language-vs-programming-quot-into-quot-a-language/) F# 😄 

---

<sup>*Means, they're almost entirely written using [object programming](https://twitter.com/dsyme/status/843849405212098560) rather than functional programmin O.o</sup>

Reply:
@moodmosaic Just noticed that you haven't approved the PR - do you have some other concerns about it?